src/operator/api/v1alpha3/otterize_labels.go

```diff
@@ -23,15 +31,16 @@ func IsMissingOtterizeAccessLabels(pod *v1.Pod, otterizeAccessLabels map[string]
 
 // UpdateOtterizeAccessLabels updates a pod's labels with Otterize labels representing their intents
 // The pod is also labeled with "otterize-client=<hashed-client-name>" to mark it as having intents or being the client-side of an egress netpol
-func UpdateOtterizeAccessLabels(pod *v1.Pod, serviceName string, otterizeAccessLabels map[string]string) *v1.Pod {
+func UpdateOtterizeAccessLabels(pod *v1.Pod, serviceIdentity serviceidentity.ServiceIdentity, otterizeAccessLabels map[string]string) *v1.Pod {
 	if pod.Labels == nil {
 		pod.Labels = make(map[string]string)
 	}
+	// TODO: We should understand what kind
```

Comment:
What is this referring to?

Reply:
I guess you wanted to re-use labels from the Service/Deployment/etc and currently don't?

Reply:
Not really, this function is about access labels (so it's not a reuse situation). The comment is probably there from an early stage of the development.